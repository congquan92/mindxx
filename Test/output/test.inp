You're just like my favorite song going round and round my head
Like my favorite song going round and round my head
You're just like my favorite song going round and round my head
Like my favorite song going round and round my head