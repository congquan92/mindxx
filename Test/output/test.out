So dong : 4
tu do xuat hien : 8 lan
max_w : favorite dai 8 . 
6 4 4 2 8 4 5 5 3 5 2 5 
4 2 8 4 5 5 3 5 2 5 
6 4 4 2 8 4 5 5 3 5 2 5 
4 2 8 4 5 5 3 5 2 4 
Hang 1 : 12 cau 
Hang 2 : 10 cau 
Hang 3 : 12 cau 
Hang 4 : 10 cau 
You're  head

Like  head

You're  head

Like  head
